libmime/mime_types.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2018 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+// This file is auto-generated and should not be edited by hand.
+
+package libmime
+
+var mimeTypes = map[string]string{
```

Comment:
Would it make sense to alphabetize this, if it's generated by a script we wrote?  (Also, if we wrote the script to generate it, should that be checked in too?)